mobile/src/styles/AppStylesheet.ts

```diff
@@ -0,0 +1,92 @@
+import {StyleSheet} from 'react-native';
+
+export const AppStylesheet = StyleSheet.create({
+    container: {
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "white",
+    },
+
+    containerSplashScreen: {
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#E5E5E5",
+    },
+
+    button: {
+        backgroundColor: "#96C957",
+        width: 327,
+        height: 52,
+        borderRadius: 3,
+        alignItems: "center",
+        justifyContent: "center",
+        marginTop: 20,
+    },
+    
+    buttonText: {
+        position: "absolute",
+        fontSize: 18,
+        fontStyle: "normal",
+        fontWeight: "normal",
+    },
+
+    ForgotPassword: {
+        width: 114,
+        height: 18,
+        marginTop: 20,
+    },
+    
+    ForgotPasswordText: {
+        fontStyle: "normal",
+        fontWeight: "normal",
+        fontSize: 15,    
+        textAlign: 'left',
+        textDecorationLine: "underline",
+    },
+    
+    input: {
+        padding: 10,
+    },
+
+    textInputField: {
```

Comment:
Just a very slight nitpicky styling thing since the figma text field has horizontal margins spacing the "Example padding" text away from the edge line. Perhaps add some padding because, currently, the words are on the edge of the texts field.
![image](https://user-images.githubusercontent.com/88344254/154158284-89641672-9ced-477b-8c93-2ef8a17d59dc.png)

Reply:
This is a good point. In the app currently the "Example Padding" text rides right up against the left edge of the TextInput. We can fix this by adding some padding to the stylesheet on the TextInput.

Reply:
done